### ISSUE TITLE
ADA issues might be resolved with v3.0 upcoming of universalviewer

### ISSUE THREAD

**Issue by verbalhanglider** [uchicago-library/universalviewer#6]
See the demonstration of the uv application here http://universalviewer.io/uv.html?manifest=https://digital.library.villanova.edu/Item/vudl:92879/Manifest#?c=0&m=0&s=0&cv=0&xywh=-1843%2C-343%2C8464%2C6843

It appears that they are in the process of updating their  documentation for v3.0
See [this  link](https://github.com/UniversalViewer/universalviewer/wiki/V3)

If you do a WAVE accessibility check there is one error: document language missing.

I will figure out how to deploy version 3.0 on iiif-viewer.lib.uchicago.edu to be able to verify this theory next.
